app/routes/events/components/EventAdministrate/AttendeeElements.tsx

```diff
@@ -1,8 +1,7 @@
+import { LoadingIndicator } from '@webkom/lego-bricks';
 import cx from 'classnames';
-import Button from 'app/components/Button';
```

Comment:
I sometimes wonder what linting actually does in this project :/:
Lines 48 and 53 could perhaps use this component for unified styling

Reply:
Hahah, not much when we keep ignoring all the linter warnings..